.github/workflows/universalnrp-test.yml

```diff
@@ -3,127 +3,120 @@ name: Universal NRP Test
 on:
   workflow_dispatch:
     inputs:
-      policyPackageList:
+      policy_packages:
         description: 'List of policy packages to test'
         required: true
         default: '[]'
   pull_request:
   schedule:
     - cron: '0 20 * * *' # Every day at 12pm PST (UTC-8)
 
-env:
-  # Default build matrix - this is where additional distros are added, policy-package are defined either through the workflow_dispatch input or the DEFAULT_PACKAGE_LIST
-  BUILD_MATRIX: |
```

Comment:
I prefer having the configurable knobs (distros/packages/arch) available at the top of the workflow as workflow globals, why move them below? Its easier to manage this way, at least that was my intent when i did it :)

Reply:
The policy packages are only relevant to the MC test workflow so I moved them into that env so that they do not show up on the env of every job/step.

I'd prefer if it wasn't JSON but since we have 2 jobs sharing the same set of targets it makes sense to have a shared variable. I'll move some stuff around to make the targets shared between the 2 jobs too. :)